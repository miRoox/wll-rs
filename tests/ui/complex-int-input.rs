use wll::{Complex, Result};

#[wll::setup]
fn setup() {}

#[wll::teardown]
fn teardown() {}

#[wll::export(conj)]
fn cconj(z: Complex<i64>) -> Result<Complex<i64>> {
    Ok(z.conj())
}

fn main() {}
