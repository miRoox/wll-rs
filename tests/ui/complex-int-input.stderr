error[E0599]: no function or associated item named `try_get_arg` found for struct `Complex<i64>` in the current scope
  --> $DIR/complex-int-input.rs:10:13
   |
10 | fn cconj(z: Complex<i64>) -> Result<Complex<i64>> {
   |             ^^^^^^^^^^^^ function or associated item not found in `Complex<i64>`
